src/Form/AgeCheckerForm.php

```diff
@@ -0,0 +1,143 @@
+        ),
+      );
+    }
+
+    $form['age_checker_error_message'] = array(
+      '#type' => 'markup',
+      '#markup' => '<div id="age_checker_error_message"> </div>',
+      '#weight' => 0,
+    );
+
+    // Day form Element.
+    $form['day'] = array(
+      '#type' => 'textfield',
+      '#size' => 2,
+      '#maxlength' => 2,
+      '#id' => 'age_checker_day',
+      '#weight' => \Drupal::state()->get('age_checker_' . $selected_country . '_day_weight'),
+      '#required' => TRUE,
+      '#attributes' => array(
+        'pattern' => "[0-9]*",
+        'tabindex' => \Drupal::state()->get('age_checker_' . $selected_country . '_day_weight'),
+        'placeholder' => \Drupal::state()->get('age_checker_' . $selected_country . '_day_placeholder'),
+      ),
+    );
+
+    // Month form Element.
+    $form['month'] = array(
+      '#type' => 'textfield',
+      '#size' => 2,
+      '#maxlength' => 2,
+      '#id' => 'age_checker_month',
+      '#required' => TRUE,
+      '#weight' => \Drupal::state()->get('age_checker_' . $selected_country . '_month_weight'),
+      '#attributes' => array(
+        'pattern' => "[0-9]*",
+        'tabindex' => \Drupal::state()->get('age_checker_' . $selected_country . '_month_weight'),
+        'placeholder' => \Drupal::state()->get('age_checker_' . $selected_country . '_month_placeholder'),
+      ),
+    );
+
+    // Year form Element.
+    $form['year'] = array(
+      '#type' => 'textfield',
+      '#size' => 4,
+      '#maxlength' => 4,
+      '#id' => 'age_checker_year',
+      '#weight' => \Drupal::state()->get('age_checker_' . $selected_country . '_year_weight'),
+      '#required' => TRUE,
+      '#attributes' => array(
+        'pattern' => "[0-9]*",
+        'tabindex' => \Drupal::state()->get('age_checker_' . $selected_country . '_year_weight'),
+        'placeholder' => \Drupal::state()->get('age_checker_' . $selected_country . '_year_placeholder'),
+      ),
+    );
+
+    // Remember Me Checkbox.
+    $option_remember_me = \Drupal::state()->get('age_checker_option_remember_me');
+    if ($option_remember_me == 1) {
+      $form['remember_me'] = array(
+        '#type' => 'checkbox',
+        '#weight' => 5,
+        '#id' => 'age_checker_remember_me',
+        '#title' => \Drupal::state()->get('age_checker_' . $language_code . '_remember_me_text'),
+        '#default_value' => 0,
+        '#attributes' => array(
+          'tabindex' => '5',
+        ),
+      );
+    }
+
+    // Submit button.
+    $form['submit'] = array(
+      '#type' => 'submit',
+      '#value' => \Drupal::state()->get('age_checker_' . $language_code . '_button_text'),
+      '#weight' => 6,
+      '#attributes' => array(
+        'onclick' => "age_checker.verify();",
+        'tabindex' => '6',
+      ),
+    );
+
+    $form['#attributes']['onsubmit'] = 'return false;';
+
+    return $form;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
```

Comment:
@purushotamrai  This is not a configuration form. This is the age gate checker form, where values entered are validated by js and  it is not required to save any value.